localstack/services/sns/sns_listener.py

```diff
@@ -375,6 +375,16 @@ def message_to_subscribers(message_id, message, topic_arn, req_data, subscriptio
                 LOG.warning('Unable to forward SNS message to SNS platform app: %s %s' % (exc, traceback.format_exc()))
                 sns_error_to_dead_letter_queue(subscriber['SubscriptionArn'], req_data, str(exc))
 
+        elif subscriber['Protocol'] == 'email':
+            ses_client = aws_stack.connect_to_service('ses')
+            if subscriber.get('Endpoint'):
+                ses_client.verify_email_address(EmailAddress=subscriber.get('Endpoint'))
+                ses_client.verify_email_address(EmailAddress='admin@localstack.com')
```

Comment:
nitpick: better put this hard-coded source email address into a global constant like `SES_SENDER_EMAIL` (shouldn't hold back the merge, can be done in a future iteration.)